test/unit/lib/plugins/aws/package/lib/stripNullPropsFromTemplateResources.test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../utils/run-serverless');
+
+const expect = chai.expect;
+
+describe('stripNullPropsFromTemplateResources', () => {
```

Comment:
I think we didn't add that to documentation, but we adopted a convention where top-level describes should use the path to the file as the name, so in this case it would be `test/unit/lib/plugins/aws/package/lib/stripNullPropsFromTemplateResources.test.js`